### DISCUSSION TITLE
Using tmpfs avoids disks TBW?

### DISCUSSION THREAD

**Discussion by cayolblake** [madMAx43v3r/chia-plotter#559] [Q&A]
If I use tmpfs for temp 1 and 2, does this mean no temp IO on the disk occurs unless it’s doing the final write?

**Comment by cyperbg** [comment]
If you use tmpfs for Temp 1 and 2 then no nvme is involved at all, so no writes to it obviously.

**Reply by cayolblake** [reply]
So how many Terabytes written for temp 1 and temp 2 separately?
